neo/UInt256.cs

```diff
@@ -32,16 +32,19 @@ public UInt256(byte[] value)
         /// Method CompareTo returns 1 if this UInt256 is bigger than other UInt256; -1 if it's smaller; 0 if it's equals
         /// Example: assume this is 01ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00a4, this.CompareTo(02ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00a3) returns 1
         /// </summary>
-        public int CompareTo(UInt256 other)
+        public unsafe int CompareTo(UInt256 other)
         {
-            byte[] x = ToArray();
-            byte[] y = other.ToArray();
-            for (int i = x.Length - 1; i >= 0; i--)
+            fixed (byte* px = ToArray(), py = other.ToArray())
             {
-                if (x[i] > y[i])
-                    return 1;
-                if (x[i] < y[i])
-                    return -1;
+                ulong* lpx = (ulong*)px;
+                ulong* lpy = (ulong*)py;
+                for (int i = 4; i >= 0; i--)
+                {
```

Comment:
i = 4 or x.Lenght ?

Reply:
in this case it should be i=8
Since 256bit = 32bit * 8

Reply:
Yeah, it should be i=8.  We need unit tests before safely accepting this, and clearly benchmarking the change (leaving a record for the future).

Reply:
I tried i=8, it looks to have some error in some rare cases... I don't know exactly. Let's merge benchmarking structure first, so we are sure this works.

Reply:
Actuallt should be i=7 since it includes 0

Reply:
ulong = 64bit，ulong*4 =256bit

uint =32bit，uint*5 =160bit

Reply:
Missed it was ulong; you are correct, but it is still wrong, it should be i=3 then, since it includes 0.

Reply:
Also, I updated @igormcoelho benchmark to include a benchmark using ulong, but it is still not faster than the original implementation.